hello-api/src/main/scala/com/example/hello/api/HelloService.scala

```diff
@@ -18,7 +18,7 @@ trait HelloService extends Service {
 
   override final def descriptor = {
     import Service._
-    named("hello")
+    named("helloworld.GreeterService")
```

Comment:
If descriptor name and gRPC service name match, then service location _just works_ ™️